docs/user-guide/generate-certificates.md

```diff
@@ -102,21 +91,79 @@ Because `--update-config` was specified the `zowe.certificates` section's values
       certificateAuthorities: /global/zowe/keystore/local_ca/local_ca.cer
 ```
 
-When using a self-signed certificate, you will be challenged by your browser when logging in to Zowe to accept its untrusted certificate authority.  Depending on the browser you are using there are different ways to proceed.  
+When using a self-signed certificate, you will be challenged by your browser when logging in to Zowe to accept its untrusted certificate authority.  Depending on the browser you are using there are different ways to proceed.
 
-### Manually import a certificate authority into a web browser
+## Create a self signed JCERACFKS certificate
 
-To avoid the browser untrusted CA challenge, you can import Zowe's certificates into the browser to avoid untrusted network traffic challenges. For more information, see [Import the local CA certificate to your browser](../extend/extend-apiml/certificate-management-in-zowe-apiml.md/#import-the-local-ca-certificate-to-your-browser).
+The following `zowe.yaml` example will generate:
 
-To avoid requiring each browser to trust the CA that signed the Zowe certificate, you can use a public certificate authority such as _Symantec_, _Comodo_, _Let's Encrypt_, or _GoDaddy_to create a certificate. These certificates are trusted by all browsers and most REST API clients. This option, however, requires a manual process to request a certificate and may incur a cost payable to the publicly trusted CA.
+ - A `JCERACFKS` certificate, specified in `zowe.setup.certificate.type` 
+ - A keyring named `ZoweKeyring` specified in  `zowe.setup.certificate.keyring.name`. 
+ - A certificate with the label `localhost` specified in `zowe.setup.certificate.keyring.label`  
+ - A certificate authority with the label `localca` specified in  `zowe.setup.certificate.keyring.caLabel` with a common name `Zowe Service CA`.
 
-<!--
+```
+zowe:
+  setup:
+    certificate:
+      # Type of certificate storage. Valid values are: PKCS12 or JCERACFKS
+      type: JCERACFKS
+      keyring:
+        name: Zowe Keyring
+        label: localhost
+        caLabel: localca
+      dname:
+        caCommonName: Zowe Service CA
+        commonName:
+        orgUnit:
+        org:
+        locality:
+        state:
+        country:
+      validity: 3650
+```
 
-## Import an existing self signed PKCS12 certificate
+The follow command output shows generation of a self signed JCERACFKS certificate using the default values.  Some detailed output messages have been omitted.
 
-**TODO**
+When the command is run a customized JCL member name in created the `CUST.JCLLIB` data set.  The PDS name is defined in the `zowe.setup.dataset.jcllib` property.  In the sample below the PDS meember `USER.ZWEV2.CUST.JCLLIB(ZW101431)` is created that contains the security manager commands and then submitted as a job ID `ZWEKRING(JOB03054)`.  
```

Comment:
I think you need a comma after "When the command is run" and "In the sample below"